etc/base.yaml

```diff
@@ -226,7 +226,7 @@ radar_appserver_postgresql:
 
 radar_appserver:
   _install: false
-  _chart_version: 0.1.4
+  _chart_version: 0.1.6
```

Comment:
appserver needs a new helm chart release for latest app version (v2.3.0) to be compatible with MP 2.0.

Reply:
Thanks! It's fixed now